src/fuse.js

```diff
@@ -336,6 +336,19 @@
     keys: []
   };
 
+  Fuse.prototype.add = function(to_add) {
+    if (!Utils.isArray(to_add)) {
+      to_add = [to_add];
+    }
+    this.list = this.list.concat(to_add);
```

Comment:
Perhaps do this instead:

`Array.prototype.push.apply(this.list, Utils.isArray(to_add) ? to_add : [to_add])`